modules/dnn/include/opencv2/dnn/utils/inference_engine.hpp

```diff
@@ -40,7 +40,7 @@ CV_EXPORTS_W cv::String setInferenceEngineBackendType(const cv::String& newBacke
  * Single Myriad device cannot be shared across multiple processes which uses
  * Inference Engine's Myriad plugin.
  */
-CV_EXPORTS_W void resetMyriadDevice();
+CV_EXPORTS_W void resetMyriadDevice(const std::string& device = "MYRIAD");
```

Comment:
@alalek, can you suggest a new version number for `#define OPENCV_DNN_API_VERSION 20200908`?

Reply:
I would prefer to add dedicated `resetHDDLDevice()` API for that.